packages/core/email/admin/src/index.ts

```diff
@@ -9,8 +9,10 @@ import { prefixPluginTranslations } from '@strapi/helper-plugin';
 
 import { PERMISSIONS } from './constants';
 
+// eslint-disable-next-line import/no-default-export
 export default {
-  register(app) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  register(app: any) {
```

Comment:
Could you add a TODO here? We should probably prioritize typing `app` because every package needs it. Could you create an issue for the next sprint for that?